bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifCodec.java

```diff
@@ -59,6 +60,22 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
         }
     }
 
+    @Override
+    public void userEventTriggered(@Nullable ChannelHandlerContext ctx, @Nullable Object evt) throws Exception {
+        if (ctx == null) {
+            return;
+        }
+        if (evt instanceof IdleStateEvent) {
+            IdleStateEvent e = (IdleStateEvent) evt;
+            logger.trace("IdleStateEvent received {}", e.state());
+            onvifConnection.setIsConnected(false);
+            // }
+            ctx.close();
+        } else {
+            logger.trace("other event received {}", evt);
```

Comment:
Could this be a little more descriptive like "Other ONVIF netty channel event occured {}" as Netty is also used for the main camera comms and a separate set for ONVIF.

Reply:
Good idea.